### ISSUE TITLE
When using remote data directory in JBrowse desktop, allow saving session still

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse#1269]

**Comment by scottcain** [comment]
Is there a technology issue that prevents this from happening?

**Comment by cmdcolin** [comment]
Not really, it's mostly organizational that is based on pointing to a remote jbrowse instance that cooperates. If it is based strictly on pointing the electron instance using the "?data=http://remote_instance/" then there are also some limitations such as

a) this method doesn't load that instances jbrowse.conf/jbrowse_conf.json
b) allowCrossOriginDataRoot tries to prevent this for sort of good reasons

**Comment by cmdcolin** [comment]
The main thing needed is some buy-in that just makes reliable things for the jbrowse desktop instance to point to (or more likely, jbrowse.org can keep a list of reliable mod's to connect to that jbrowse desktop just asks a list of)

**Comment by scottcain** [comment]
Ah, I wasn't planning on doing "data=http://...." rather having a local trackList.json that points at remote data in the urlTemplate, so perhaps this issue doesn't apply to what I want to do with electron.

**Comment by cmdcolin** [comment]
Oh. Sorry I didn't realize the issue you were posting on was not the other sort of related one (#1251) so my comments might be a little weird

This bug is related to the concept specifically of "Open remote data directory" e.g. the ?data=http://remote/thing/" and it doesn't create any sort of local copy of the tracklist.json in that case so then the idea of saving a session doesn't work.

If you have a pre-loaded-actual-data-directory with remote urlTemplates that comes with jbrowse desktop then this bug won't exist.
